test/README.md

```diff
@@ -1,5 +1,63 @@
 ## End-to-End Testing
 
+### AWS E2E Testing
+Featured in this directory is a `./awsE2ESolutionSetup` script whose goal is to encapsulate deploying an E2E environment in AWS. This script will perform the following key actions:
+1. Deploy an Elasticsearch source cluster to EC2 using the following [CDK](https://github.com/lewijacn/opensearch-cluster-cdk/tree/migration-es)
+   * The CDK context options for this CDK can be used to customize this deployment and can be provided with a custom context file using the `--context-file` option
+   * A sample CDK context file can be seen [here](defaultCDKContext.json)
+2. Build the Docker Images
+3. Deploy the Migration Assistant CDK (containing the Migration tooling and potentially an OpenSearch Domain) located in this repo [here](../deployment/cdk/opensearch-service-migration)
+   * The CDK context options for this CDK can be used to customize this deployment and can be provided with a custom context file using the `--context-file` option
+   * A sample CDK context file can be seen [here](defaultCDKContext.json)
+4. Add the traffic stream source security group created by the Migration Assistant CDK to the source cluster load balancer
```

Comment:
Another sentence or two here explaining what this means would be appreciated by someone doing this for the first time, I think.  Mind adding that?

Reply:
Good mention, have added and updated some incorrect wording here